scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -13,7 +13,8 @@ import scala.meta.internal.{semanticdb => s}
 import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
 import scala.meta.internal.semanticdb.Scala.{Names => n}
-import scalafix.internal.v0._
+import scala.meta.internal.semanticdb.{Scala, Signature, TypeRef, TypeSignature}
```

Comment:
The idiomatic way of referring to `Signature` etc is to `import scala.meta.internal.{semanticdb => s}` (already done above) and then say `s.Signature` etc.

Reply:
Thanks. This is IJ auto-imports. I forgot about your renaming imports strategy but will revise.